hikari/api/entity_factory.py

```diff
@@ -1777,6 +1777,23 @@ def deserialize_template(self, payload: data_binding.JSONObject) -> template_mod
     # USER MODELS #
     ###############
 
+    @abc.abstractmethod
+    def deserialize_avatar_decoration(
```

Comment:
There will be no point when we will need to deserialize the avatar only, so this can be made a private method and removed from here